<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Parameter_CurveProgression_Form</class>
 <widget class="QWidget" name="Parameter_CurveProgression_Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1025</width>
    <height>675</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1025</width>
    <height>675</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#Parameter_CurveProgression_Form
{
	border-image: url(:/resource/images/common/Transparent_Black_70.png);
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>92</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>307</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>91</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>307</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QWidget" name="widgetBg" native="true">
     <property name="minimumSize">
      <size>
       <width>800</width>
       <height>480</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>800</width>
       <height>480</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <widget class="PushButtonDone" name="pushButton">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>400</y>
        <width>112</width>
        <height>82</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Nexa Light</family>
        <pointsize>26</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
	border-image: url(:/resource/images/upload_setting/Done.png);
	border-color: rgba(255, 255, 255, 0);
}
QPushButton:hover
{
	border-image: url(:/resource/images/upload_setting/Done_OVER.png);
	border-color: rgba(255, 255, 255, 0);
}</string>
      </property>
      <property name="text">
       <string>Done</string>
      </property>
      <property name="shortcut">
       <string>Ctrl+Return</string>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="CurveProgressionWidget" name="curveBG" native="true">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>761</width>
        <height>381</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/resource/images/parameter_input/CurveProgression/CurveProgression_BG.png);</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="currentPosY">
      <property name="geometry">
       <rect>
        <x>447</x>
        <y>426</y>
        <width>62</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 8pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="suffix">
       <string>%</string>
      </property>
      <property name="decimals">
       <number>1</number>
      </property>
      <property name="maximum">
       <double>100.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="currentPosX">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>426</y>
        <width>62</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 8pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="suffix">
       <string>%</string>
      </property>
      <property name="decimals">
       <number>1</number>
      </property>
      <property name="maximum">
       <double>100.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="currentPosLabel_2">
      <property name="geometry">
       <rect>
        <x>289</x>
        <y>429</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>8</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 8pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>Current Postion:</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PushButtonDone</class>
   <extends>QPushButton</extends>
   <header>PushButtonDone.h</header>
  </customwidget>
  <customwidget>
   <class>CurveProgressionWidget</class>
   <extends>QWidget</extends>
   <header>CurveProgressionWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
